### PR TITLE
<Thumbnail /> enhancements Property to override default placeholders

### PR DESCRIPTION
**Pull request by skvale**
<Thumbnail /> enhancements Property to override default "waiting" and "not available" SVG graphics.

### CODE REVIEW

File: src/components/thumbnail/index.jsx [hunk 1]
```diff
@@ -14,7 +14,9 @@ class Thumbnail extends Component {
     static propTypes = {
         id: PropTypes.number.isRequired,
         maxSize: PropTypes.number,
-        uploader: PropTypes.object.isRequired
+        uploader: PropTypes.object.isRequired,
+        notAvailablePlaceholder: PropTypes.element,
```
Comment (rnicholus) [inline src/components/thumbnail/index.jsx:17]: Can these be ABC ordered? I'm trying to do this throughout the library.

**Comment by rnicholus** [comment]
Other than the small re-ordering adjustment, can you update the README.md file too? After that, I think this will be ready to merge. Thanks for you help!

P.S. Don't worry about the failing build. This is happening due to an auth error with SauceLabs, since it is running against your forked repo (which is not tied to the encrypted access key).

**Comment by skvale** [comment]
Great, 
One thing which may be confusing is if you set `maxSize` it won't have any affect on the custom SVGs.
Thanks Ray

**Comment by rnicholus** [comment]
Oh, I forgot about `maxSize`. It seems like the custom image should be scaled by the component/library too.

File: src/components/thumbnail/placeholder.jsx [hunk 2]
```diff
@@ -3,7 +3,8 @@ import React, { PropTypes } from 'react'
 const Placeholder = ({ image, size, status }) => {
     const style = {
         maxHeight: size,
-        maxWidth: size
+        maxWidth: size,
+        overflow: 'hidden'
```
Comment (skvale) [inline src/components/thumbnail/placeholder.jsx]: Maybe this isn't appropriate, but it does enforce the maxSize.

Comment (rnicholus) [inline src/components/thumbnail/placeholder.jsx]: I forgot about the existing `max-width` and `max-height` styles. Overflow shouldn't be necessary.

**Pull request merged**